onnxruntime/test/python/quantization/test_quantize_static.py

```diff
@@ -89,6 +90,46 @@ def test_save_as_external(self):
             check_model_correctness(self, self._model_fp32_path, quant_model_path, data_reader.get_next())
             data_reader.rewind()
 
+    def test_stride_effect_on_data_collection(self):
```

Comment:
Thank you for adding unit tests 👍 

Here are some instructions on how to run these tests locally (just in case you haven't done so already):

#### 1. Build onnxruntime from source:
```shell
python .\tools\ci_build\build.py --cmake_generator "Visual Studio 17 2022" --config Release --build_shared_lib --build_wheel --skip_tests --build_dir build --parallel
```

#### 2. Install local python wheel in build\Release\Release\dist
```shell
cd build\Release\Release\dist
pip uninstall onnxruntime
pip install onnxruntime-1.19.0-cp310-cp310-win_amd64.whl
```

#### 3. Run unit tests
```shell
cd onnxruntime\test\python\quantization
```
To run **all** quantization unit tests:
```shell
python -m unittest
```

To run unit tests in a specific file:
```shell
python -m unittest test_quantize_static.py
```